integrations.json

```diff
@@ -612,6 +612,86 @@
       }
     ]
   },
+  {
```

Comment:
Please remember to re-generate this file before merging, as we are in version 1.26.3 now